gms/gossiper.cc

```diff
@@ -52,7 +52,6 @@ using clk = gossiper::clk;
 static logging::logger logger("gossip");
```

Comment:
Patch description has to be more verbose, something like:

```
gms/gossiper: store expire time in a member variable

In the next patches we will make this configurable. To prepare for that, store it in a regular member, instead of a static constant.
```

Note that the above example:
* mentions the changed module and file in the subject
* subject contains a short description of what happens
* the body adds further details
* subject + body describes what changes and the motivation of the change

Please follow this pattern for reach commit.